rules/framework.bzl

```diff
@@ -337,6 +337,7 @@ apple_framework_packaging = rule(
         ),
         "deps": attr.label_list(
             mandatory = True,
+            cfg = apple_common.multi_arch_split,
```

Comment:
From the [docs](https://docs.bazel.build/versions/master/skylark/lib/apple_common.html)

```
Use of this transition requires that the 'platform_type' and 'minimum_os_version' string attributes are defined and mandatory on the rule.
```
I see that these attributes are not mandatory here, is that intentional?

Reply:
Yep, it's mandatory that these are available on the rule. We might need to have different defaults than what Bazel is applying in some situations, will look more into it

Reply:
I see, I misunderstood the docs then. I thought that the "mandatory" attr for them should be "True" but they just need to be present then.